test/mainController.test.ts

```diff
@@ -199,21 +204,21 @@ suite('MainController Tests', () => {
 
     test('onNewQuery should call the new query and new connection' , () => {
 
-        untitledSqlDocumentService.setup(x => x.newQuery()).returns(() => Promise.resolve(true));
+        untitledSqlDocumentService.setup(x => x.newQuery()).returns(() => Promise.resolve(undefined));
```

Comment:
can we add some tests for these changes?

Reply:
Yes, please add some tests

---
In reply to: [305093888](https://github.com/microsoft/vscode-mssql/pull/1259#discussion_r305093888) [](ancestors = 305093888)

Reply:
I'm writing tests now, I haven't completed the PR yet. The next iteration will have them :)